### DISCUSSION TITLE
Relations in many to many

### DISCUSSION THREAD

**Discussion by wjarochowicz91** [mikro-orm/mikro-orm#1692] [Q&A]
So i have User entity with m : m relation to book, and that book have m:1 relation to category, and now if im loading User, and looking at his book, everything is just fine, but i want category also and the problem is im getting only something like 
`
Ref<CategoryEntity> { id: '776475f2-14ff-4c93-939e-e65cab156c0b' }
`
Is there any other way to load it directly ? Or it's the thing that i need to load it manually using repo or something else ?

**Comment by B4nan** [comment]
You need to populate the relation. If you want more help, show some code.
